### ISSUE TITLE
Load antibiotic pack-description dataset to store app and push to data collector app

### ISSUE THREAD

**Issue by mverzilli** [instedd/maap-collector#13]

**Comment by hdf1986** [comment]
@mverzilli Is this going to be like in #4 ?

**Comment by mverzilli** [comment]
Yes, it will have similar mechanics. But instead of being a list of "scalar values", it is going to be a more complex entity (see Pharmaceutical Product in the ERD diagram: https://www.draw.io/?state=%7B%22ids%22:%5B%221PrWYfvNxjcvzg1WOHjfXfjdGAaywhxIv%22%5D,%22action%22:%22open%22,%22userId%22:%22110185406749669396797%22%7D#G1PrWYfvNxjcvzg1WOHjfXfjdGAaywhxIv)

Basically, each antibiotic pack-description will have:

- A reference to the list of antibiotics
- Name: string
- Brand: string
- Molecule: string
- Pack size: string
- Grammage: string
- ATC 5: string
- Manufacturer: string
- Form: string
- MRP per pack: string
